spanfiltering/transpile_test.go

```diff
@@ -86,17 +97,26 @@ func TestTranspileFilter(t *testing.T) {
 				return
 			}
 			assert.NilError(t, err)
-			actual, err := TranspileFilter(filter)
+			actual, params, err := TranspileFilter(filter)
 			if err != nil && tt.errorContains != "" {
 				assert.ErrorContains(t, err, tt.errorContains)
 				return
 			}
 			assert.NilError(t, err)
 			assert.Equal(t, tt.expectedSQL, actual.SQL())
+			assert.DeepEqual(t, tt.expectedParams, params)
 		})
 	}
 }
 
+func mustParseTime(s string) time.Time {
```

Comment:
make this a t.Helper() and use assert instead of panic?

Reply:
fixed!